packages/editor/src/store/selectors.js

```diff
@@ -134,6 +134,65 @@ export const isEditedPostDirty = createRegistrySelector( ( select ) => ( state )
 	return false;
 } );
 
+/**
+ * Returns true if there are unsaved edits for entities other than
+ * the editor's post, and false otherwise.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {boolean} Whether there are edits or not.
+ */
+export const hasNonPostEntityChanges = createRegistrySelector(
```

Comment:
It would be good to have some unit tests for this selector. And also the one in the other file (`getEntityRecordChangesByRecord`).